docs/gui/cult-lists.rst

```diff
@@ -0,0 +1,58 @@
+cult-lists.lua
```

Comment:
It kind of feels like this script could be the home for all religion-related functionality. how about calling it `gui/religion`?